AzureTable/Microsoft.DataTransfer.AzureTable/ConfigurationResources.resx

```diff
@@ -132,4 +132,7 @@
   <data name="Source_Table" xml:space="preserve">
     <value>Name of the Azure Table</value>
   </data>
+  <data name="Source_LocationMode" xml:space="preserve">
+    <value>Optional, choose which location mode to use</value>
```

Comment:
I think this should include to place holder: one for default value and another one for the list of all available values. Take a look at "Source_InternalFieldsFormat", for example. It seems like the code that returns the description does account for them to be here.